examples/third_party/python/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+load("@rules_python//python:defs.bzl", "py_runtime", "py_runtime_pair", "py_test")
+
+exports_files(
+    [
+        "BUILD.python.bazel",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+py_runtime(
+    name = "py3_runtime",
+    files = ["@python"],
```

Comment:
Could you instead use the `gen_dir` output group to create a filegroup with all of these files? If using `gen_dir` includes too much you can always use the `postfix_script` attribute to remove any files you don't need from the install directory?
If that works then this would avoid the need to add an extra attribute.

Reply:
This also seems related to https://github.com/bazelbuild/rules_foreign_cc/issues/582

Reply:
@jsharpe  I'll look into using `gen_dir`

Reply:
@jsharpe  I tried adding the following `filegroup` to `Build.python3.bazel`

```
filegroup(
    name = "gen_dir",
    srcs = [":python3"],
    output_group = "gen_dir",
)
```

I then set the `files` attribute of the `py_runtime` to `["@python3//:gen_dir]`. Sadly the files inside the `lib` directory generated by the build of python were not included in the `runfiles` for the `python3_test`. As such, the invocation of `python3` failed.

Perhaps I am doing something wrong?